cli/internal/config/config.go

```diff
@@ -10,62 +10,50 @@ import (
 	"runtime"
 	"strings"
 
-	"github.com/vercel/turborepo/cli/internal/client"
-
-	hclog "github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/go-hclog"
 	"github.com/kelseyhightower/envconfig"
 	"github.com/mattn/go-isatty"
 	"github.com/mitchellh/cli"
+	"github.com/vercel/turborepo/cli/internal/client"
 )
 
 const (
 	// EnvLogLevel is the environment log level
 	EnvLogLevel = "TURBO_LOG_LEVEL"
 )
 
-// IsCI returns true if running in a CI/CD environment
 func IsCI() bool {
 	return !isatty.IsTerminal(os.Stdout.Fd()) || os.Getenv("CI") != ""
 }
 
-// Config is a struct that contains user inputs and our logger
 type Config struct {
```

Comment:
As a general comment on this file, it does not appear that we need the changes in this file. If they are necessary for a later PR, can you reintroduce them then?

Reply:
These changes don't appear necessary to move `bin` to `cobra`.